plugins/tracer.py

```diff
@@ -64,50 +67,57 @@ def transaction(self):
                          self.base.transaction.install_set])
         erased = set([package.name for package in
                       self.base.transaction.remove_set])
+        packages = [Package(p, time.time()) for p in list(installed | erased)]
 
-        args = ["tracer", "-n"] + list(installed | erased)
-        process = subprocess.Popen(
-            args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-        out, err = process.communicate()
-        _print_output(out, err)
+        try:
+            tracer = TracerFacade(packages)
+            tracer.render()
 
-        if len(out) != 0:
-            print("\n" + _("For more information run:"))
-            print("    sudo tracer -iat " + str(self.timestamp))
+            if len(tracer.apps) != 0:
+                print("\n" + _("For more information run:"))
```

Comment:
You mean getting list of applications equivalent to `tracer -iat <timestamp>`? Not yet, I can implement it. But how do you want to utilize it in the plugin?